frontend/src/features/splReports/columns.tsx

```diff
@@ -0,0 +1,152 @@
+import { ISnapshot } from './interfaces';
+import { CellProps } from 'react-table';
+import { formatMoney } from 'utils';
+
+const howManyColumns = 13;
+const totalWidthPercent = 100; // how wide the table should be; e.g. 100%
+
+// Setup a few sample widths: x/2, 1x, 2x (percentage-based)
+const unit = Math.floor(totalWidthPercent / howManyColumns);
+const spacing = {
+  xxsmall: 1,
+  xsmall: unit / 4,
+  small: unit / 2,
+  medium: unit,
+  large: unit * 2,
+  xlarge: unit * 4,
+  xxlarge: unit * 8,
+};
+
+export const columns: any[] = [
+  {
+    Header: 'SPP No.',
+    accessor: 'project.projectNumber', // accessor is the "key" in the data
+    align: 'left',
+    responsive: true,
+    width: spacing.small,
+    minWidth: 65, // px
+  },
+  {
+    Header: 'Fiscal Year',
+    accessor: 'project.actualFiscalYear',
+    align: 'left',
+    responsive: false,
+    width: 50,
+    minWidth: 50,
+  },
+  {
+    Header: 'Agency',
+    accessor: (row: ISnapshot) => row?.project?.agency ?? row?.project?.subAgency ?? '',
+    align: 'left',
+    responsive: false,
+    width: 50,
+    minWidth: 50,
+  },
+  {
+    Header: 'Name',
+    accessor: 'project.name',
+    align: 'left',
+    responsive: true,
+    width: spacing.medium,
+    minWidth: 80,
+  },
+  {
+    Header: 'CMV',
+    accessor: 'estimated',
+    align: 'left',
```

Comment:
Shouldn't financial values alight right?  Below there are a number of instances of this